task/06-conditions-n-loops-tasks.js

```diff
@@ -192,7 +240,23 @@ function findFirstSingleChar(str) {
  *
  */
 function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
-    throw new Error('Not implemented');
+    let bracket_first;
+    let bracket_last;
+
+    if (isStartIncluded === true) {
```

Comment:
`if (isStartIncluded) {` this will work
Please, read about https://developer.mozilla.org/en-US/docs/Glossary/Truthy